.circleci/config.yml

```diff
@@ -3,23 +3,15 @@ jobs:
   build:
     working_directory: ~/mesos_state
     docker:
-      - image: erlang:20.1
+      - image: erlang:20
```

Comment:
why are we going back?

Reply:
`erlang:20` is an alias for the latest release of OTP 20.y.z.